buildenv/jenkins/triggerGrinder.groovy

```diff
@@ -1,46 +1,49 @@
 #!groovy
 
-def json_path = args.size() ? args[0] : "disabledTestParser/output.json."
-def json = readJSON file: json_path
-def jenkins_url = "https://ci.adoptopenjdk.net/job/Grinder/buildWithParameters"
+List<Map<String, Object>> json = []
+
+node {
+  checkout scm
+  stage('readJSON') {
+    def json_path = "${WORKSPACE}/aqa-tests/disabledTestParser/output.json"
+    json = readJSON(file: json_path) as List<Map<String, Object>>
+  }
+  stage('Launch Grinder Jobs') {
+    launch_grinders(json)
+  }
+}
+
+def launch_grinders(List<Map<String, Object>> json) {
+  /**
+   Get each of the key-value pairs for each json value.
+   If the issue for a job is closed, run grinder on it.
+   Otherwise ignore it
+   */
+  json.eachWithIndex { dict, _ ->
+    if (dict["ISSUE_TRACKER_STATUS"] == "closed") {
+      run_grinder(dict)
+    }
+  }
+}
 
 /**
  This runs when we have found a job w/ git_issue_status = closed.
  Collects all parameters and runs grinder
  */
-def run_grinder(map) {
-  def list = []
-  def queryString
+def run_grinder(Map<String, Object> map) {
+  def childParams = []
 
   map.each { k, v ->
-    list.add(k + "=" + v)
+    if (k != "ISSUE_TRACKER_STATUS" && k != "ISSUE_TRACKER") {
+      childParams << string(name: "${k}", value: "${v}")
+    }
   }
 
-  queryString = list.join("&")
-  url = jenkins_url + "/?" + queryString
-
-  // The Grinder trigger will look something like this,
-  // but it requires tokens which I do not have yet.
-  // def url = new URL(jenkins_url + "/?" + queryString)
-  // def connection = url.openConnection()
-  // connection.with {
-  //     doOutput = true
-  //     requestMethod = 'POST'
-  //     outputStream.withWriter { writer ->
-  //         writer << queryString
-  //     }
-  //     println content.text
-  // }
-
-  println url
-}
-
-/**
- Get each of the key-value pairs for each json value.
- If the issue for a job is closed, run grinder on it.
- Otherwise ignore it */
-json.eachWithIndex { dict, _ ->
-  if (dict["GIT_ISSUE_STATUS"] == "closed") {
-    run_grinder(dict)
+  for (e in childParams) {
+    println e
   }
+
+  def job = build job: "Grinder", parameters: childParams, propagate: false
```

Comment:
Could you change it to `propagate: true`? And I think we do not need lines 47 and 48.